package.json

```diff
@@ -34,7 +34,9 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config jest-e2e.json && chronix test",
+    "test:e2e": "docker-compose --env-file=./.env -f docker-compose.e2e.yml up --build --abort-on-container-exit",
+    "test:e2e:docker": "mikro-orm schema:drop -r && mikro-orm migration:up && jest -i --config jest-e2e.json && chronix test",
```

Comment:
why do we name it test:e2e:docker >

Reply:
"test:e2e:docker": "mikro-orm schema:drop -r && mikro-orm migration:up && jest -i --config jest-e2e.json && chronix test   - is a test running command 

docker-compose --env-file=./.env -f docker-compose.e2e.yml up --build --abort-on-container-exit - running in container

Reply:
we can name it `test:e2e:ci`, WDYT?